src/commands/chains.ts

```diff
@@ -14,5 +14,5 @@ export const builder: CommandBuilder<{}, Options> = (yargs) =>
 const chains = getChains();
 
 export const handler = async (_argv: Arguments<Options>): Promise<void> => {
-  console.dir(chains, { depth: null });
+  console.log(JSON.stringify(chains, null, "\t"));
```

Comment:
I think we should just render raw JSON (not indented), then let people use `jq` to make it pretty. That's a more flex pattern for people wanting to pipe to files, etc.